# Use vcs plugin instead of versioningit for hatch version. Not only is the former packaged for Alpine, it allows
# overriding the version using the SETUPTOOLS_SCM_PRETEND_VERSION variable instead of needing to manually patch a file.
diff --git i/pyproject.toml w/pyproject.toml
index 0ce149c..cd11cf8 100644
--- i/pyproject.toml
+++ w/pyproject.toml
@@ -66,7 +66,7 @@ docs = [
     "mkdocs-git-revision-date-localized-plugin>=1.4.7",
 ]
 [tool.hatch.version]
-source = "versioningit"
+source = "vcs"

 [tool.hatch.build.targets.wheel]
 include = [
